css-font-display/index.bs

```diff
@@ -276,6 +277,22 @@ for example, forcing all fonts to have a ''0s'' <a>block period</a>.
 		</div>
 </dl>
 
+Controlling Font Display Per Font-Family via ''@font-feature-values''
+===========================
+The '@font-feature-values/font-display' descriptor for ''@font-feature-values'' determines how a font family is displayed, by setting the "default" font-display value for @font-face rules targeting the same font family.
+When font-display is omitted in an @font-face rule, the user agent uses the font-display value set via the @font-feature-values/font-display for the relevant font-family if one is set, and otherwise defaults to "font-display: auto".
+
+This mechanism can be used to set a default display policy for an entire font-family, and enables developers to set a display policy for @font-face rules that are not directly under their control.
+For example, when a font is served by a third-party font foundry, the developer does not control the @font-face rules but is still able to set a default font-display policy for the provided font-family.
+The ability to set a default policy for an entire font-family is also useful to avoid the ransom note effect (i.e. mismatched font faces) because the display policy is then applied to the entire font family.
+
+<pre class='descdef'>
+Name: font-display
+Value: auto | block | swap | fallback | optional
+Initial: auto
+For: @font-feature-values
+</pre>
+
 <div class='issue'>
```

Comment:
No, this is to support the "served by a third party" use case and the ability to set a font-display across a font-family instead of per font-face (the ransom note issue).

Is the issue about the names not being limpid still current?
If it is then I could send a proposal via a PR or discuss options over an issue.